content/docs/graphql/read-only-tokens.md

```diff
@@ -0,0 +1,217 @@
+---
+title: Read-only Tokens
+last_edited: '2022-02-07T18:00:00.000Z'
+---
+{{ WarningCallout text="This is an experimental feature and may be slow as we work on performance improvements" }}
+
+Read-only tokens allow data fetching at runtime without the need for the local graphQL server. Some use cases include the following:
+
+
+- Runtime server-side logic in `getServerSideProps`, `getStaticProps` (when fallback is not `false`), etc.
+- [Incremental Static Site Generation](https://nextjs.org/docs/basic-features/data-fetching/incremental-static-regeneration)
+- [Server components](https://nextjs.org/docs/advanced-features/react-18#react-server-components)
```

Comment:
```suggestion
- [Server components](https://nextjs.org/docs/advanced-features/react-18/overview#react-server-components-alpha)
```
seems to link to the server components heading, the other just goes to the same page but at the top